tests/Controller/ExportControllerTest.php

```diff
@@ -180,4 +181,36 @@ protected function parsePluginVersion(): string
 
         return ltrim($parsed['version'], 'v');
     }
+
+    public function testCorrectTranslatedProductIsExported(): void
+    {
+        $anotherLanguageId = Uuid::randomHex();
+        $this->createLanguage($anotherLanguageId);
+        $this->salesChannelContext = $this->buildSalesChannelContext(
+            Defaults::SALES_CHANNEL,
+            'http://test.com',
+            null,
+            $anotherLanguageId
+        );
+
+        $expectedName = 'test translated';
+        $product = $this->createVisibleTestProduct(
+            [
+                'translations' => [
+                    ['name' => $expectedName, 'languageId' => $anotherLanguageId],
+                ],
+            ]
+        );
+
+        $this->enableFindologicPlugin($this->getContainer(), self::VALID_SHOPKEY, $this->salesChannelContext);
+
+        $response = $this->sendExportRequest(['productId' => $product->getId()]);
```

Comment:
Send another request to ensure that if there are two shopkeys for different language, each shopkey will return the product in the proper translation.

DE: `ABCDABCDABCDABCDABCDABCDABCDABCD` => Product name: `Produkt`
EN: `1BCDABCDABCDABCDABCDABCDABCDABCD` => Product name: `Product`